app/scripts/modules/core/src/modal/wizard/WizardModal.tsx

```diff
@@ -62,19 +63,25 @@ export class WizardModal<T = {}> extends React.Component<IWizardModalProps<T>, I
     return this.formikRef.current && (this.formikRef.current.getFormikBag() as FormikProps<T>);
   }
 
+  public componentDidMount(): void {
+    this.setState({ initialized: true });
+  }
+
   public onWizardPageAdded = (wizardPage: WizardPage<T>): void => {
     this.setState(prevState => {
       const pages = prevState.pages.concat(wizardPage);
       const currentPage = prevState.currentPage || pages[0];
-      return { pages, currentPage };
+      const initialized = prevState.initialized;
```

Comment:
this shouldn't be necessary (`setState()` with the same value of `initialized`)

Reply:
👍